apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -2045,6 +2045,12 @@ data class FeatureEnabled(
 sealed class ContactUserPref {
   @Serializable @SerialName("contact") data class Contact(val preference: ChatPreference): ContactUserPref() // contact override is set
   @Serializable @SerialName("user") data class User(val preference: ChatPreference): ContactUserPref() // global user default is used
+
+  val pref: ChatPreference
+    get() = when(this) {
+      is Contact -> preference
+      is User -> preference
+    }
```

Comment:
Possibly, you can just use .preference because it is present in both subclasses of sealed class.

If not, it should be defined as an abstract property on superclass level and overridden in subclasses, there is no need to have this computed property at all.